suite-common/icons/src/components/CoinIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { networks, NetworkSymbol } from '@suite-common/wallet-config';
+import { TokenSymbol } from '@suite-common/wallet-types';
+
+import { CryptoIcon, CryptoIconSize } from './CryptoIcon';
+import { TokenIcon } from './TokenIcon';
+import { CryptoIconName } from '../icons';
+
+export type CoinSymbol = CryptoIconName | TokenSymbol;
+
+type CoinIconProps = {
+    symbol: CoinSymbol;
+    size?: CryptoIconSize;
+};
+
+export const CoinIcon = ({ symbol, size }: CoinIconProps) => {
```

Comment:
Can we keep only rename this to CryptoIcon and basically merge tokens into it? I don't see reason why not and current state is really confusing. You may keep these components internally but they shouldn't be exposed out of package and probably should be grouped in folder or placed in CryptoIcon file directly.

Reply:
Ok, I understood from the original PR where was `TokenIcon` introduced, that we want to have the network icons and token icons separated for easier further improvements. But I agree that this is way better from a developer experience perspective so I've reworked it to make only CryptoIcon exposed (2981f6c5f).